modules/dynamic-plug-in-development.adoc

```diff
@@ -1,32 +1,41 @@
 // Module included in the following assemblies:
 //
-// * web_console/dynamic-plug-ins.adoc
+// * web_console/dynamic-plug-ins-get-started.adoc
 
 :_content-type: PROCEDURE
-[id="running-your-dynamic-plugin_{context}"]
-= Running your dynamic plug-in
+[id="dynamic-plugin-development_{context}"]
+= Dynamic plug-in development
 
 You can run the plug-in using a local development environment. The {product-title} web console runs in a container connected to the cluster you have logged into.
 
 .Prerequisites
-* You must have the OpenShift CLI (`oc`) installed.
 * You must have an OpenShift cluster running.
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have link:https://yarnpkg.com/[`yarn`] installed.
 * You must have link:https://www.docker.com/[Docker] or at least v3.2.0 of link:https://podman.io/[Podman] installed.
 
 .Procedure
 
-. Build a plug-in and generate the output to the `dist` directory by running
+. In your terminal, run the following command to start yarn.
+//This is probably the wrong procedure step, but wanted to fill it in with a placeholder.
++
+[source,terminal]
+----
+yarn run start
+----
+
+. Build a plug-in and generate the output to the `dist` directory by running the following command:
 +
 [source,terminal]
 ----
-$ yarn build
+yarn build
 ----
 
 . Start an HTTP server by running
 +
 [source,terminal]
 ----
-$ yarn http-server
+yarn http-server
```

Comment:
This is wrong if you're using the plugin template. `yarn run start` is correct.

It looks like we've mistakenly copied things from the demo plugin in the console repo which is not the same as the template. We should roll back any of those changes.